benchmarks/graph_infra/arkouda.graph

```diff
@@ -22,6 +22,24 @@ files: groupby.dat, groupby.dat, groupby.dat, groupby.dat
 graphtitle: Groupby Performance
 ylabel: Performance (GiB/S)
 
+perfkeys: Aggregate sum Average rate =, Aggregate xor Average rate =, Aggregate mean Average rate =
+graphkeys: sum (GiB/s), xor (GiB/s), mean (GiB/s)
+files: aggregate.dat, aggregate.dat, aggregate.dat
+graphtitle: Grouped Aggregation Performance (Tier 1)
```

Comment:
Do the tiers indicates which ones you care most about, or is it somewhat arbitrary just to avoid putting everything on one graph?

Reply:
The tiers are by speed, so that slow ops won't get lost in the noise floor when graphed alongside fast ops. I put them into tiers based on the rates I observed on a single locale, so these tiers might be inappropriate for multi-locale.

Reply:
Gotcha. I like the idea of separating them like this, but wonder if there's a term that makes it a little more obvious that it's just speed tiers? In some of the chapel perf graphs we have 2 tiers labeled `(faster idioms)` and `(slower idioms)`. Maybe fastest/faster/slower here? I don't feel very strongly about this, so I'm fine leaving it as-is too.